.gitignore

```
@@ -3,3 +3,4 @@
 docker-compose.yml
 docker-compose-dynamic.yaml
 docker-compose-dynamic-add.yaml
+**/.venv
```

`**/` seems superfluous here, I suggest we use the recommended setup for python repository: https://github.com/github/gitignore/blob/master/Python.gitignore#L102 
```suggestion
.venv
```